cpp/src/io/json/write_json.cu

```diff
@@ -135,81 +134,51 @@ struct escape_strings_fn {
   }
 };
 
-// Struct concatenation.
-struct concat_structs_base {
-  table_device_view const d_table;
-  column_device_view const d_column_names;
-  string_view const row_prefix;             //{
-  string_view const row_suffix;             //} or }\n for json-lines
-  string_view const d_col_separator;        //:
-  string_view const d_val_separator;        //,
-  string_scalar_device_view const d_narep;  // null
-  bool include_nulls = false;
-  offset_type* d_offsets{};
-  char* d_chars{};
+// Struct - scatter string_views of each element in a struct column
+struct struct_scatter_strings_fn {
+  table_device_view const tbl;
+  column_device_view const col_names;
+  size_type const strviews_per_column;
+  size_type const num_strviews_per_row;
+  string_view const row_prefix;       //{
+  string_view const row_suffix;       //} or }\n for json-lines
+  string_view const value_separator;  //,
```

Comment:
```suggestion
  string_view const row_prefix;       // {
  string_view const row_suffix;       // } or }\n for json-lines
  string_view const value_separator;  // ,
```

Reply:
How is this not caught in clang-format?

Reply:
Seems like it should. I noticed the new clang-format doing some odd things myself.

Reply:
We noticed a bug in clang-format 16.0.1 relating to spacing around `*` (it made some multiplications look like pointers). We can revert to another version if you think there are issues with the version we've chosen.

Reply:
I've not seen anything concerning that is worth reverting to an older version.